.github/workflows/lint.yaml

```diff
@@ -19,28 +19,28 @@ jobs:
       - name: Check for secrets
         run: gitleaks detect --verbose --redact
 
-  pip:
-    name: Pip
+  uv:
+    name: uv
```

Comment:
It bothers me that word uv cannot be capitalised like the others otherwise it just looks weird 😂